src/main/java/org/jitsi_modified/impl/neomedia/rtp/remotebitrateestimator/OveruseEstimator.java

```diff
@@ -0,0 +1,321 @@
+/*
+ * Copyright @ 2015 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi_modified.impl.neomedia.rtp.remotebitrateestimator;
+
+import org.jetbrains.annotations.*;
+import org.jitsi.utils.logging.*;
+
+import java.util.*;
+
+/**
+ * webrtc/modules/remote_bitrate_estimator/overuse_estimator.cc
+ * webrtc/modules/remote_bitrate_estimator/overuse_estimator.h
+ *
+ * @author Lyubomir Marinov
+ */
+class OveruseEstimator
+{
+    /**
+     * The <tt>Logger</tt> used by the
+     * <tt>RemoteBitrateEstimatorAbsSendTime</tt> class and its instances for
+     * logging output.
+     */
+    private static final Logger logger
```

Comment:
a bit torn on doing a pure import vs making some changes, but i do think it'd be nice to change all these to use the new logger

Reply:
Pushed a change to use the new logger.